sickbeard/showUpdater.py

```diff
@@ -92,43 +92,43 @@ def run(self, force=False):  # pylint: disable=unused-argument, too-many-locals,
         except SyntaxError:
             pass
 
-        logger.log(u"Doing full update on all shows")
+        logger.log(u'Doing full update on all shows')
```

Comment:
@Thraxis I think its because you are doing a logger.log instead of for example logger.info and you arent passing a logging level.